spec/lib/whitehall_importer/integrity_checker/body_text_check_spec.rb

```diff
@@ -15,27 +15,55 @@
       expect(integrity_check.sufficiently_similar?).to be true
     end
 
-    it "returns true even if there is a mismatch in inline atttachment URL filesize" do
+    it "returns true even if there is a mismatch in a linked inline atttachment URL filesize" do
```

Comment:
Whoops!  Will fix.  The only need for `a linked` is to distinguish between the two tests.  Otherwise, apart from the body text's they are the same, but testing one with - and one without - an anchor tag.  Hence, linked.  I'll see if I can come up with something more meaningful.